src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImplTest.java

```diff
@@ -15,73 +16,115 @@
 import edu.cornell.kfs.concur.batch.fixture.ConcurRequestExtractFileFixture;
+import edu.cornell.kfs.concur.batch.report.ConcurRequestExtractBatchReportData;
 import edu.cornell.kfs.concur.batch.service.ConcurBatchUtilityService;
 import edu.cornell.kfs.sys.CUKFSConstants;
 import edu.cornell.kfs.sys.CUKFSParameterKeyConstants;
 
 public class ConcurRequestExtractFileValidationServiceImplTest {
+    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractFileValidationServiceImplTest.class);
+
     private ConcurRequestExtractFileValidationServiceImpl concurRequestExtractFileValidationService;
     private ConcurParameterConstantsFixture concurParameterConstantsFixture;
     private ConcurBatchUtilityServiceImpl concurBatchUtilityService;
+    private ConcurApplicationPropertiesFixture concurApplicationPropertiesFixture;
+    private TestableConfigurationServiceImpl configurationService;
     
     @Before
     public void setUp() throws Exception {
         Logger.getLogger(ConcurRequestExtractFileValidationServiceImpl.class).setLevel(Level.DEBUG);
         concurRequestExtractFileValidationService = new ConcurRequestExtractFileValidationServiceImpl();
         concurBatchUtilityService = new TestableConcurBatchUtilityServiceImpl();
         concurRequestExtractFileValidationService.setConcurBatchUtilityService(concurBatchUtilityService);
+        configurationService = new TestableConfigurationServiceImpl();
+        concurRequestExtractFileValidationService.setConfigurationService(configurationService);
         concurParameterConstantsFixture = new ConcurParameterConstantsFixture();
+        concurApplicationPropertiesFixture = new ConcurApplicationPropertiesFixture();
     }
     
     @After
     public void tearDown() throws Exception {
         concurRequestExtractFileValidationService = null;
         concurBatchUtilityService = null;
         concurParameterConstantsFixture = null;
+        concurApplicationPropertiesFixture = null;
+        configurationService = null;
     }
     
     @Test
     public void testHeaderAmountAndHeaderRowCountsMatch() {
+        LOG.info("testHeaderAmountAndHeaderRowCountsMatch");
         ConcurRequestExtractFile testFile = ConcurRequestExtractFileFixture.GOOD_FILE.createConcurRequestExtractFile();
-        assertTrue("Expected Result: Header amount SHOULD match sum of row amounts from file.", concurRequestExtractFileValidationService.requestExtractHeaderRowValidatesToFileContents(testFile));
+        ConcurRequestExtractBatchReportData reportData = new ConcurRequestExtractBatchReportData();
+        assertTrue("Expected Result: Header amount SHOULD match sum of row amounts from file.", concurRequestExtractFileValidationService.requestExtractHeaderRowValidatesToFileContents(testFile, reportData));
+        LOG.info(reportData.getHeaderValidationErrors());
+        System.out.println("");
```

Comment:
It gives white space when you run the unit tests so that it is easier to see each unit test in its entirety. I could change it to a LOG.info(""): instead if you prefer.